languages/javascript/exercises/concept/booleans/.docs/instructions.md

```diff
@@ -4,10 +4,10 @@ After some time spent following her best friend's trail, she finds the camp in w
 
 Having found the kidnappers, Annalyn considers which of the following actions she can engage in:
 
-- *Fast attack*: a fast attack can be made if the knight is sleeping, as it takes time for him to get his armor on, so he will be vulnerable.
-- *Spy*: the group can be spied upon if at least one of them is awake. Otherwise, spying is a waste of time.
-- *Signal prisoner*: the prisoner can be signalled using bird sounds if the prisoner is awake and the archer is sleeping, as archers are trained in bird signaling so they could intercept the message.
-- *Free prisoner*: Annalyn can try sneaking into the camp to free the prisoner, but this tactic will only work if the prisoner is awake and the other two characters are sleeping. If the prisoner is sleeping, they'll be startled by Annalyn's sudden appearance and will awaken the other two characters. The prisoner can also be freed if the archer is sleeping and Annalyn has her pet dog with her, as the knight will be scared by the dog and will withdraw, and the archer can't equip his bow fast enough to prevent the prisoner from being freed.
+- _Fast attack_: a fast attack can be made if the knight is sleeping, as it takes time for him to get his armor on, so he will be vulnerable.
```

Comment:
I disagree with this change. I think these should be bold - that'll make them much clearer to the eye when parsign these instructions.